### DISCUSSION TITLE
[Enhancement] [transfer] 全选性能优化

### DISCUSSION THREAD

**Discussion by QianTangrong** [element-plus/element-plus#10132] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `transfer`

## Description

现在全选功能在数据量稍微大点都会有明显卡顿，大概在1k条数据左右就会卡顿明显，吃满cpu，能否优化下性能

### Additional comments

_(empty)_

**Comment by emojiiii** [comment]
这应该要加个虚拟列表。
